kata-webhook/main.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	whhttp "github.com/slok/kubewebhook/pkg/http"
+	"github.com/slok/kubewebhook/pkg/log"
+	mutatingwh "github.com/slok/kubewebhook/pkg/webhook/mutating"
+)
+
+func annotatePodMutator(_ context.Context, obj metav1.Object) (bool, error) {
+	pod, ok := obj.(*corev1.Pod)
+	if !ok {
+		// If not a pod just continue the mutation chain(if there is one) and don't do nothing.
+		return false, nil
+	}
+
+	//We cannot really support --net=host in Kata
```

Comment:
You could add a ref to https://github.com/kata-containers/documentation/blob/master/Limitations.md#docker---nethost